trymito.io/styles/[slug].module.css

```diff
@@ -102,11 +124,75 @@
   border-bottom: 1.5px solid black; 
 }
 
+.post_toc {
+    width: 265px;
+    min-width: 265px;
+    height: fit-content;
+    position: sticky;
+    top: 67px;
+    margin-left: 30px;
+}
+
 .post_content table thead {
     background-color: black;
     border-radius: 1rem;
 }
 
+.post_cta {
+    width: 265px;
```

Comment:
These should also be in rem. Take a look at some of the other css files too. 

The other reason that this is important is because our font size changes between mobile and desktop. So basing it off of rem means the width and spacing will be proportional to the font still